modules.php

```diff
@@ -44,5 +44,13 @@
 		$modules[] = ( require "$modules_dir/ppcp-googlepay/module.php" )();
 	}
 
+	if ( apply_filters(
+		//phpcs:disable WordPress.NamingConventions.ValidHookName.UseUnderscores
+		'woocommerce.deprecated-flags.woocommerce_paypal_payments.saved_payment_checker_enabled',
```

Comment:
Is there an agreement about this naming? Why not just use `woocommerce.deprecated_flags` with `_`?